docs/implementation-guide/sizing.md

```diff
@@ -1,101 +1,98 @@
+
+!!! note "1GbE only recommended for External Network communication"
+
+### Controller Nodes (Node 1 and Node 2)
+
+- 1GB of RAM per 1TB of storage for each node
+- 1 x 800GB+ 3 DWPD NVMe-based SSD for vSAN metadata
+- 5GB of vSAN metadata storage per 1TB of usable capacity in your VergeOS System
+
+### Storage Nodes (Nodes that partipate in vSAN storage)
+
+- 1GB of RAM per 1TB of storage for each node
+- Minimum of one NVMe or SATA/SAS SSD per node for workload storage
+- HDDs can be used for snapshot, archive, or file based service storage
+- At least 2 nodes with the same disk configuration for data redundancy
+
+!!! warning "HDDs larger than 8TB are not recommended in non-archive-specific environments due to extended drive rebuild times."
+
+### Compute-only Nodes
+
+- Only the [generic requirements](#generic-requirements-all-node-types) apply to Compute-only nodes
+- CPU and Memory should be architected for workloads running on these nodes
 
 ### GPU (Optional)
-**vGPU Requirements:**
 
-  - Processor with IOMMU support (Intel VT-d, AMD-Vi/VT).
+- Compatible with any AMD, Intel, or Nvidia GPU
+- Nvidia vGPU also supported
 
-  - Refer to the Hardware Compatibility List (HCL) for tested drivers and GPUs.
+<!-- ### Additional Network Considerations
 
-**GPU Passthrough:**
+- For environments with more than two servers switches are required for Core Fabric Networks
+- Core Fabric Networks require jumbo frames of 9216 MTU or greater -->
 
-  - Compatible with any AMD, Intel, or Nvidia GPU.
-  - Nvidia vGPU also supported
+## Recommended Requirements
 
----
+### Generic Requirements (All Node Types) - Recommended
+
+- Intel or AMD x86 64-bit processor with hardware virtualization support
+- Minimum of 16GB RAM dedicated to VergeOS
+- HBA (preferred) or RAID controller that supports JBOD mode - NO RAID
+- Minimum of 16GB RAM dedicated to VergeOS
+- 2 x 25/40GbE NICs (Intel, Mellanox, or Broadcom)
+- 2 x 10/25GbE NICs (Intel, Mellanox, or Broadcom)
+- **Dedicated Controller Nodes**
+
+### Controller Nodes (Node 1 and Node 2) - Recommended
 
-## Recomneded Requirements
+- 1 x +3.0Ghz CPU clockrate
+- 2 x 800GB+ 3 DWPD NVMe-based SSD for vSAN metadata (Tier 0)
+- 5GB of storage per 1TB of usable capacity for vSAN metadata (Tier 0)
 
-### CPU
-- **vSAN (Storage Nodes)**: Intel Xeon or AMD EPYC Processor, 8c – 16t Processor 3.0 GHz
-- **Compute Nodes**: Intel Xeon or AMD EPYC Processor, Dual 32 core Processors 2.2 GHz
-- Processor family must be consistent across the cluster
-- Must support VT-x (for Intel) or AMD-V/AMD-SVM (for AMD)
+### Storage Nodes (Nodes that partipate in vSAN storage) - Recommended
 
-### Memory
-- Minimum of 16 GB dedicated to VergeOS for OS and application server operations.
-- **Compute Nodes**: Memory to Core ratio of 8-1/16-1
-- **Storage Nodes**: 1.5GB per 1TB of RAW storage
+- 2 x +3.0Ghz CPU clockrate
+- 1 CPU core per disk
+- 1GB of RAM per 1TB of storage for each node
```

Comment:
being recommended, we would want more ram per raw TB of disk to allow for more caching 1.5 or 2gb per 1TB raw